src/commands/force/org/display.ts

```diff
@@ -27,6 +27,13 @@ export class OrgDisplayCommand extends SfdxCommand {
   };
 
   public async run(): Promise<OrgDisplayReturn> {
+    try {
+      // the auth file might have a stale access token.  We want to refresh it before getting the fields
+      await this.org.refreshAuth();
```

Comment:
What if the access token is not stale? Any way to check before forcing the auth?

Reply:
That's already built in with refreshAuth().  It's definitely only updating them if they went stale...I assume it's trying to use them, catching expirations, and using the refreshToken (where available) or re-jwt-ing.

Reply:
Correct. It tries to use it and if the server says it is invalid or expired, jsforce will kick off the refresh flow that we defined.